packages/x-components/static-docs/experience-search-&-discovery/web-local-storage.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I don't see this related to the rest of the components doc. Should it be in another place?

Reply:
I've discussed this before with @herrardo. For sure, it's not a doc related to the UI doc, but a transversal doc that won't have direct access from the content menu in the portal. It can be accessed from different docs that refer to local browser history. If it happens to create more docs alike, we can consider the option of moving it to a specific subfolder. Can we consider the idea of just moving it to the root folder?

Reply:
I think we can leave it here as is going to be referred from documents in this folder. At least by now.